Test/PerformanceCollector/FunctionalTests/Serialization/TelemetryExtensions.cs

```diff
@@ -40,6 +41,30 @@ public static class TelemetryExtensions
             return result;
         }
 
+        internal static MonitoringDataPoint[] ReceiveItems(
+          this QuickPulseHttpListenerObservable listener,
+          int count,
+          int timeOut)
+        {
+            if (null == listener)
+            {
+                throw new ArgumentNullException("listener");
+            }
+
+            var result = listener
```

Comment:
nitpick: I'm guessing "result" is of type "MonitoringDataPoint[]" but the var declaration doesn't give me any readability.